includes/class-wc-calypso-bridge-ecommerce-admin-menu.php

```diff
@@ -292,7 +295,7 @@ public function add_woocommerce_menu() {
 			);
 		}
 
-		if ( class_exists( '\Automattic\WooCommerce\Admin\Features\Features' ) && \Automattic\WooCommerce\Admin\Features\Features::is_enabled( 'analytics' ) ) {
+		if ( class_exists( '\Automattic\WooCommerce\Utilities\FeaturesUtil' ) && \Automattic\WooCommerce\Utilities\FeaturesUtil::feature_is_enabled( 'analytics' ) ) {
```

Comment:
I'm not 100% clear if the new (p7bje6-4kh-p2) and [old](https://github.com/woocommerce/woocommerce/blob/9ea83d6382580d106e818dd0cff7c48e8e173359/plugins/woocommerce-admin/docs/features/feature-flags.md) feature declarations are compatible with each other. I don't see any examples of the `analytics` feature using the `FeaturesUtil` method, only `Features::is_enabled` calls.

Reply:
Great catch! Indeed, this isn't working with the `analytics` feature! Reverted https://github.com/Automattic/wc-calypso-bridge/pull/1318/commits/976722331efd87836d53437ae352a1374876d295. Thanks 🙌